README.md

```diff
@@ -5,6 +5,14 @@ Common typings for the Stoplight ecosystem.
 - Explore the interfaces: [TSDoc](https://stoplightio.github.io/types/)
 - View the changelog: [Releases](https://github.com/stoplightio/types/releases)
 
+### Dependent repositories (not an exhaustive list):
```

Comment:
https://github.com/stoplightio/types/network/dependents

Reply:
@P0lip, thanks. Didn't know about that. So, there a lot more repos to be updated because of the change (not only `json-ref-resolver`, `spectral`, `http-spec`, `prism`, `graphite`). So, afther the bump in `types`, do I update all the repos with that bumped `types` version?

Reply:
You don't necessarily need to unless the change you introduce affects all of them.
I don't think you need to update json or yaml for instance as they don't make use of that interface. That said, I think the list of repos you planned to update should be sufficient.